prac_08/convert_miles_km.kv

```diff
@@ -0,0 +1,25 @@
+BoxLayout:
+    orientation: "vertical"
+    BoxLayout:
+        TextInput:
+            id: input_field
+            text: ""
+            on_text: app.handle_conversion(input_field.text)
+        BoxLayout:
+            orientation: "vertical"
+            Button:
+                id: up_button
+                text: "Up"
+                on_press: app.handle_increment(input_field, 1)
+            Button:
+                id: down_button
+                text: "Down"
+                on_press: app.handle_increment(input_field, -1)
```

Comment:
I see a bit of repetition within the handle_increment parameters. Is there a better way to do this?

Reply:
Good pickup.  Stemming from the problem in the app code discussed below 😃